supplier/tests/test_supplier_page.py

```diff
@@ -39,3 +40,41 @@ def test_supplier_check_false(non_supplier_mock):
 def test_supplier_check_true(saved_supplier0):
     assert saved_supplier0 in Supplier.objects.all()
     assert get_supplier(saved_supplier0) == saved_supplier0
+
+
+@pytest.mark.django_db()
+def test_supplier_deliveries(client, delivery_location0):
+    delivery_location0.add_delivery_location()
+    client.force_login(user=delivery_location0.user_name.supplier_account)
+    response = client.get('/suppliers/')
+    assert list(response.context['supplier_deliveries']) == [delivery_location0]
```

Comment:
why do we cast both sides to a list?

Reply:
left side returns QueryList and deliverylocation is an delivery object,the result we want [A]=[A] so on the left side we cast it to list and right side we just create a list that contants the deliverylocation

Reply:
why not compare the first item of the QueryList (or the casted list, if we absolutely must) to the `deliverylocation`? let's change the result we expect as little as possible.

Reply:
(You don't *have* to change it, just something to think about)

Reply:
But what if the function was wrong and it returned [A,B] in this case it will pass but it actually didnt do what we really wanted

Reply:
upon reading the test again, you do have a good point there. a possibility is to check the length of the returned list, but we shouldn't do that in the current PR